Layout w HTML i CSS/warmup/position/style.css

```diff
@@ -0,0 +1,34 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.box {
+    background-color: lightblue;
+    height: 240px;
+    width: 240px;
+    position: absolute;
+}
+
+.box-container {
+    height: 500px;
+    width: 500px;
+    position: absolute;
+    top: 50%;
+    right: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.box-2 {
+    top: 0;
+    right: 0;
+}
+
+.box-3 {
+    top: 260px;
```

Comment:
Fajne i na ten moment okay, tylko jeśli klient zmieni wymagania co do rozmiaru zewnętrznego kontenera, to trzeba będzie poprawić i rozmiar kontenera i ten parametr tak w box-3 jak i w box-4 :)

Reply:
Racja - lepiej byłoby zamienić w box-3 i box-4 top: 260px; na bottom: 0; i w box-4 dodatkowo left: 260px; na right: 0;

Dziękuję za cenną uwagę :)